_data/talks/2018.yml

```diff
@@ -222,7 +222,7 @@
   tags:
     - Hyperledger
 
-- title: "Hyperledger Framework – Bringing blockchain technologies closer to tomorrow’s businesses"
+- title: "Hyperledger Framework – Bringing blockchain technologies closer to tomorrow's businesses"
```

Comment:
Please don’t change typographic quote to programmatic ones
https://practicaltypography.com/straight-and-curly-quotes.html

Reply:
@modkaffes 
I replace it because the title has to be the same with other entry of this workshop. Otherwise when I use the `where` filter to query the data I don't get the correct results. 
Also there are more programmatic quotes rather than typographic. So I decide to replace it with programmatic ones. 
Should I change only this line back or should I fix the whole file?

Reply:
@joskar I see! No, correct it to your convenience. We’re going to do aesthetic changes afterwards anyway.